pkg/mdformatter/linktransformer/config.go

```diff
@@ -0,0 +1,48 @@
+// Copyright (c) Bartłomiej Płotka @bwplotka
+// Licensed under the Apache License 2.0.
+
+package linktransformer
+
+import (
+	"bytes"
+	"regexp"
+
+	"github.com/pkg/errors"
+	"gopkg.in/yaml.v3"
+)
+
+type Config struct {
+	Version int
+
+	Validators []Validator `yaml:"validators"`
+}
+
+type Validator struct {
+	_regex  *regexp.Regexp
+	_maxNum int
+	// Regex for type github is reponame matcher, like `bwplotka\/mdox`.
+	Regex string `yaml:"regex"`
+	// By default type is `roundtrip`. Could be `github`.
+	Type string `yaml:"type"`
```

Comment:
let's make it a `type ValidatorType string` and then provide 2 consts with those expected values (: It will be easier to use, at least in code level

Reply:
Sure! Currently, after inverting fixes, only one const is used(`roundtrip`) so, commenting the second one until it is used in future.